src/sqlancer/influxdb/InfluxDBOptions.java

```diff
@@ -0,0 +1,37 @@
+package sqlancer.influxdb;
+
+import java.util.Arrays;
+import java.util.List;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import sqlancer.DBMSSpecificOptions;
+
+@Parameters(separators = "=", commandDescription = "InfluxDB (default port: " + InfluxDBOptions.DEFAULT_PORT
+        + " default host: " + InfluxDBOptions.DEFAULT_HOST + ")")
+public class InfluxDBOptions implements DBMSSpecificOptions<InfluxDBOracleFactory> {
+    public static final String DEFAULT_HOST = "localhost";
+    public static final int DEFAULT_PORT = 8086;
+
+    @Parameter(names = "--oracle")
+    public List<InfluxDBOracleFactory> oracle = Arrays.asList(InfluxDBOracleFactory.DEFAULT);
+
+    @Parameter(names = "--username", description = "The user name used to log into InfluxDB")
```

Comment:
We have options for user name and password in the main option class, so I think we should avoid duplicating them.

https://github.com/sqlancer/sqlancer/blob/04f0b49ccbcee1054ef93f27b0055f2aa976ca05/src/sqlancer/MainOptions.java#L70